docs/automating-using-elyra.md

```diff
@@ -0,0 +1,70 @@
+# Running a Kubeflow Workflow using Elyra
+
+## Background
+
+For our data science workflows, we use Jupyter notebooks for data collection, model training and all parts of the machine learning workflow. Some of those tasks are performed sequentially and continuously, requiring us to automate the sequential running of the notebooks using a simple workflow.
+
+Using the [Elyra](https://github.com/elyra-ai/elyra) JupyterLab and the notebook pipelines visual editor, we can create and run our notebooks using Kubeflow Pipelines, which is a platform for building and deploying scalable machine learning (ML) workflows.
+
```

Comment:
would it be good to also include a link to [Kubeflow Pipelines](https://www.kubeflow.org/docs/components/pipelines/overview/pipelines-overview/)?

Reply:
@hemajv linked the websites [kubeflow pipelines doc](https://www.operate-first.cloud/users/apps/docs/kubeflow/kubeflow-pipelines/README.md) from which the above is linked.

Reply:
ah okay saw that :+1: